command/fs_list.go

```diff
@@ -0,0 +1,121 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+)
+
+type FSListCommand struct {
+	Meta
+}
+
+func (f *FSListCommand) Help() string {
+	helpText := `
+Usage: nomad fs-list [alloc-id] [path]
+
+	Displays the files in the alloc-dir of the given alloc id. The path 
```

Comment:
`List displays the files in the allocation directory for the passed allocation. The path is relative to the root of the allocation directory.`